source/_integrations/lcn.markdown

```diff
@@ -64,7 +64,7 @@ The `lcn` hardware modules and groups are represented by Home Assistant *devices
 
 The configuration of the `lcn` devices and entities is completely done using a web user interface (configuration panel).
 
-Once the integration is added to Home Assistant, you can access the `lcn` configuration panel by selecting the **Configure** button next to the respective integration entry on the [LCN integration page](https://my.home-assistant.io/redirect/integration/?domain=lcn).
+Once the integration is added to Home Assistant, you can access the `lcn` configuration panel by selecting the **Add** button next to the respective integration entry on the [LCN integration page](https://my.home-assistant.io/redirect/integration/?domain=lcn).
```

Comment:
I think this is also related to an already existing config entry, those related to the options flow?